amida-tech/indaba-client
# Feature/inba 891 jf update react toasty

Updates the react-toastify to version 4.1, removing glamor and shrinking the package. May also fix another issue we've been having with production deployments. 

#### What does this PR do?
Updates the toastify, and gets rid of glamor.

#### Related JIRA tickets:
INBA-891.

#### How should this be manually tested?
Run `yarn install`. You'll need to try a number of things to generate various pop ups. Create new projects, try adding user again, etc. Also generate errors by shutting down your greyscale deployment before doing things, as this should get you a red error toast. 

#### Background/Context
I swear, I only changed one line in main.scss. Do a diff check to confirm. 

#### Screenshots (if appropriate):
